### PR TITLE
Bridgecrew-compliance-part-2

### PR DESCRIPTION
**Pull request by mclacore**

### CODE REVIEW

File: src/main.tf [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
   name                = var.md_metadata.name_prefix
-  location            = var.vnet.specs.azure.region
   resource_group_name = azurerm_resource_group.main.name
-  dns_prefix          = "${var.md_metadata.name_prefix}-dns"
-  oidc_issuer_enabled = false
-  kubernetes_version  = var.kubernetes_version
+  location            = azurerm_resource_group.main.location
+}
+
+resource "azurerm_kubernetes_cluster" "main" {
```
[the remark this answers is not in the document]
Comment (mclacore) [reply src/main.tf:14]: Disk encryption set policy requires key vault key, which requires a key vault, which we are not currently implementing due to network firewall vulnerabilities that would exist in order to support it.

[the remark this answers is not in the document]
Comment (mclacore) [reply src/main.tf:14]: Disk encryption set policy requires key vault key, which requires a key vault, which we are not currently implementing due to network firewall vulnerabilities that would exist in order to support it.

[the remark this answers is not in the document]
Comment (chrisghill) [reply src/main.tf:15]: Is this the only way to encrypt disks, or is there default encryption (without CMK)?

File: massdriver.yaml [hunk 2]
```diff
@@ -35,78 +35,31 @@ params:
             min_size: 1
             max_size: 10
   required:
-    - kubernetes_version
+    - cluster
     - node_groups
     - observability
   properties:
-    observability:
+    cluster:
```
Comment (chrisghill) [inline massdriver.yaml:42]: Note this is a breaking change in the DB.

Currently there are 3 packages of this bundle using the public version. Two are ours, and one is a customers (but is currently unconfigured).  You will need to release this immediately, or you will need to coordinate a DB migration w/ the platform team.

Comment (mclacore) [reply massdriver.yaml:42]: I can have it tested/released by today

**Review by chrisghill — APPROVED** [review]

**Pull request merged**